pkg/gateway/model_build_targets.go

```diff
@@ -111,19 +112,38 @@ func (t *latticeTargetsModelBuildTask) buildLatticeTargets(ctx context.Context)
 		return errors.New(errmsg)
 	}
 
-	portAnnotations := undefinedPort
-	serviceExport := &mcs_api.ServiceExport{}
-	err = t.Client.Get(ctx, namespacedName, serviceExport)
-	if err != nil {
-		glog.V(6).Infof("Failed to find Service export in the DS. Name:%v, Namespace:%v - err:%s\n ", t.tgName, t.tgNamespace, err)
-	} else {
-		// TODO: Change the code to support multiple comma separated ports instead of a single port
-		//portsAnnotations := strings.Split(serviceExport.ObjectMeta.Annotations["multicluster.x-k8s.io/Ports"], ",")
-		portAnnotations, err = strconv.ParseInt(serviceExport.ObjectMeta.Annotations[portAnnotationsKey], 10, 64)
+	definedPorts := make([]int64, undefinedPort)
```

Comment:
second argument is slice size, strange to use undefinedPort as size, you can use 0 here, or shortcut to []int64{}